test/distributed/org/apache/cassandra/distributed/impl/DistributedTestSnitch.java

```diff
@@ -39,6 +40,20 @@
     private static NetworkTopology mapping = null;
     private static final Map<InetAddressAndPort, InetSocketAddress> cache = new ConcurrentHashMap<>();
     private static final Map<InetSocketAddress, InetAddressAndPort> cacheInverse = new ConcurrentHashMap<>();
+    public static volatile InetAddressAndPort sortByProximityAddressOverride;
```

Comment:
Should it be initialised explicitly to `null` for readability and consistency with other similar variables?